src/modules/0.0.7/aws_route53_hosted_zones/index.ts

```diff
@@ -21,10 +21,7 @@ export const AwsRoute53HostedZoneModule: Module2 = new Module2({
       out.parentHostedZone = await AwsRoute53HostedZoneModule.mappers.hostedZone.db.read(ctx, rrs.HostedZoneId) ??
       await AwsRoute53HostedZoneModule.mappers.hostedZone.cloud.read(ctx, rrs.HostedZoneId);
       if (!out.parentHostedZone) throw new Error('Hosted zone need to be loaded.');
-      const domainSplit = rrs.Name.split(out.parentHostedZone.domainName);
-      if (domainSplit.length > 1) {
-        out.name = domainSplit[0];
-      }
+      out.name = rrs.Name;
```

Comment:
This definitely simplifies things, but what behavior have you just removed here?